public/index.html

```diff
@@ -23,7 +23,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-  <title>ForkDelta: The alternative EtherDelta trading interface</title>
+  <meta property="og:url" content="https://forkdelta.github.io" />
```

Comment:
You should use `%PUBLIC_URL%` to get the current URL for the deployment (see line 11).